asking_for_help.md

```diff
@@ -1,8 +1,13 @@
 # How to Ask Questions
 
+## Genius Bar
+The instructors will be offering support each day during `Genius Bar`. The schedule is outlined here: [schedule](./schedule.md) and on the WDI8 Calendar: [calendar](http://ga-dc.github.io/wdi8/calendar.html)
+
+Students will be limited to `5 minutes` at a time with one-on-one instructor support during `Genius Bar`. We are doing this to ensure we can support ALL students during a session as well giving you an opportunity to try working through your problem. You may have additional time with instructors during a session after an attempt to solve your problem :)
```

Comment:
:+1:  Like it, just got to make sure instructors are on the same page, we use the timers!